### ISSUE TITLE
Suggestion: make nar (narkamaŭka) default

### ISSUE THREAD

**Issue by byg0n3** [belaudiobooks/belorthography#13]
I don't know if that was made intentionally, but right now `translit_cyr_nar_to_lat.py` uses `translit_cyr_taras_to_lat.py` under the hood.
We can argue for a long time about which one is "the correct belarusian" or "the only one", but nonetheless - narkamaŭka is more widespread, more people know it due to narkamaŭka being taught in schools and universities, it is used in all current independent belarusian media, in the end - it has clear rules and grammar.

**Comment by nbeloglazov** [comment]
It looks you **do** want to argue about which one is "the correct blearusian" :) 

But to practical issues. We were told that in order to get corrrect lacinka the original text need to be in Taraškievica. So we decided to implement two translations: Narkam => Tarask and Tarask => Lacinka. At the moment, as you saw, we don't have it. We have just a single Cyrillic => Lacinka translation, which we copied from Nasha Niva. And their version targets Narkamauka.

We are slowly working on it so the current state is not final and needs more work. Especially adding more tests that will help implementing Nar => Tarask  + ensuring that we don't introduce any regressions.
